src/cockpit/superuser.py

```diff
@@ -83,7 +84,7 @@ async def do_custom_command(self, command: str, args: Tuple, fds: List[int], std
         if command == 'cockpit.send-stderr':
             with socket.socket(fileno=fds[0]) as sock:
                 fds.pop(0)
-                socket.send_fds(sock, [b'\0'], [2])
```

Comment:
Could you keep this commented out with some "# once we can rely on Python 3.x"?

Reply:
OK.